Adds empty documents for our coding standards (#146)
kyma-project/community: guidelines/coding-standards/golang.md

# Golang Best Practices Guide

> The guide covers the Kyma Team's considered best practices for developing in Golang.

Reviewer: ```suggestion
This guide covers the best Golang development practices every Kyma team should employ. 
```
